packages/lune/postprocess.mjs

```diff
@@ -1,26 +1,9 @@
 // @ts-check
 
 import { readFile, writeFile } from "fs/promises";
-import { build } from "esbuild";
 
-await build({
-  entryPoints: ["dist/src/index.js"],
-  bundle: true,
-  outfile: "dist/clibundle.cjs",
-  external: [
-    // literally everything except from esbuild-plugin-solid
-    "chokidar",
-    "esbuild",
-    "esbuild-sass-plugin-ysink",
-    "postcss",
-    "postcss-modules",
-    "solid-js",
-    "ws",
-    // node things esbuild doesn't know about
-    "readline/promises",
-  ],
-  platform: "node",
-});
```

Comment:
We *need* to bundle the solid compiler due to npm's inability to pin deps-of-deps so this change is a no-go

Reply:
fix it then lol